### PR TITLE
Release v1.0

### PR DESCRIPTION
**Pull request by aureate-labs-team**
- Extended API layer
- Added variations support on PDP
- Handle "Product not found"
- Bug fixes
- Optimised code

### CODE REVIEW

**Comment by aureate-labs-team** [comment]
Release v1.0

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "@storefront-ui/vue": "^0.10.3",
-    "@vue-storefront/shopify": "0.1.2-rc.1",
+    "@vue-storefront/shopify": "1.0",
```
Comment (bloodf) [inline package.json:15]:
```suggestion
    "@vue-storefront/shopify": "1.0.0",
```

**Review by bloodf — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vue-storefront/shopify-theme",
-  "version": "0.1.2-rc.1",
+  "version": "1.0",
```
Comment (bloodf) [inline package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vue-storefront/shopify-theme",
-  "version": "0.1.2-rc.1",
+  "version": "1.0.0",
   "private": true,
```
Comment (bloodf) [inline package.json:4]:
```suggestion
```

**Pull request merged**